CondFormats/RunInfo/BuildFile.xml

```diff
@@ -5,6 +5,8 @@
 <use name="FWCore/Utilities"/>
 <use name="CoralBase"/>
 <use name="boost_serialization"/>
+<use name="FWCore/Framework"/>
```

Comment:
Using code from `FWCore/Framework` in data format packages (excluding tests) is a code smell.